go/vt/vtctl/vtctl.go

```diff
@@ -2967,7 +2971,7 @@ func commandWorkflow(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag.
 	if err != nil {
 		return err
 	}
-	if action == "list" || action == "listall" {
+	if action == "show" || action == "listall" {
```

Comment:
Do we not print anything out for `show` / `listall`?

Reply:
@rohit-nayak-ps this doesn't look right. However, it is unrelated to the changes in this PR so I'm going to move this forward.

Reply:
The printing happens internally actually. From what I can tell this is a way of handling split code paths from subcommands.

Reply:
As Peter replied, show/listall follows a separate path and outputs a json doc. Otherwise we just return rows affected (result of vexec for stop/start/delete workflow)